### DISCUSSION TITLE
Windows for ARM support

### DISCUSSION THREAD

**Discussion by muceua-Projects** [dorssel/usbipd-win#1021] [Q&A]
Hi there!

I'm used to using usbipd on my old laptop (x64). I bought a new laptop with a newly released Qualcomm X Elite chip (ARM64 architecture) and tried using usbipd on it, but it did not work. After installing it, I can see my devices in the list, and it looks like it can "bind" the device, but after binding, I always get a message: "service is not running. Reboot should fix the issue". But reboot does not fix it. I was never able to "attach" the device to WSL2. 

Do you have any plans to release an ARM64-compatible version? Or is there a workaround to make it work?

**Comment by dorssel** [comment]
Duplicate of https://github.com/dorssel/usbipd-win/issues/233
